code/modules/wiremod/shell/module.dm

```diff
@@ -115,6 +137,7 @@
 
 /obj/item/circuit_component/mod_adapter_core/unregister_shell(atom/movable/shell)
 	UnregisterSignal(attached_module, COMSIG_MOVABLE_MOVED)
+	UnregisterSignal(attached_module, COMSIG_MOD_TOGGLED)
```

Comment:
please make sure attached_module is valid before doing this.

Reply:
that was a leftover mistake from some tomfoolery, fixed